src/lt/plugins/doc.cljs

```diff
@@ -76,12 +76,44 @@
               (not= (:doc doc) "nil"))
      [:pre (:doc doc)])])
 
+(defn- retrieve-behavior
+  "Helper method for behavior `editor.doc.show!` to determine if the given `ns` and `name`
+  match existing behaviors.
+
+  Returns found behavior or `nil`."
+  [ns name]
+  (@object/behaviors (keyword (str ns "/" (subs name 2)))))
+
+(defn- retrieve-object-def
+  "Helper method for behavior `editor.doc.show!` to determine if the given `ns` and `name`
+  match existing object defs. Not recommended to print whole object def... use destructuring.
+
+  Returns found object def or `nil`."
+  [ns name]
+  (@object/object-defs (keyword (str ns "/" (subs name 2)))))
+
+(defn- retrieve-docstring
+  "Helper method for behavior `editor.doc.show!` that returns the docstring for a matching
+  object or behavior, if any exist, or `nil`."
+  [ns name]
+  (let [beh-doc (:doc (retrieve-behavior ns name))
```

Comment:
Applies to both:
- Would it make sense to fall back to using :desc if no :doc provided (people are lazy, and some doc is better than no doc, even though :desc is sometimes more of a end user facing thing)